.cirrus.yml

```
@@ -53,6 +53,14 @@ global_task_template: &GLOBAL_TASK_TEMPLATE
 #  install_script:
 #    - choco install python --version=3.7.7 -y
 
+task:
+  name: 'ARM  [GOAL: install]  [buster]  [unit tests, no functional tests]'
```

faf07014b9c166692ca2bab10695f00a90990027, nit:
```suggestion
  name: 'ARM  [GOAL: install]  [buster]  [unit tests in QEMU, no functional tests]'
```